Example/macOS/FormView.swift

```diff
@@ -75,6 +83,7 @@ final class FormView: NSView {
         stackView.translatesAutoresizingMaskIntoConstraints = false
         
         NSLayoutConstraint.activate([
+            titleEntry.widthAnchor.constraint(greaterThanOrEqualToConstant: 300.0),
```

Comment:
I think we should move these into constants that represent what they are and for following DRY principle but it can be done either now or in a later PR.